### PR TITLE
Add Peninsular Research Operation as an end user

### PR DESCRIPTION
**Pull request by karthikeyanVK**
added my company name as per request

_Provide a description of what has been changed_

### Checklist

- [ ] Commits are signed with Developer Certificate of Origin (DCO)
- [ ] A PR is opened to update the documentation on [our docs repo](https://github.com/kedacore/keda-docs)

Fixes #

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Great, thanks! Would you mind adding your logo and fix DCO issues please?

File: config.toml [hunk 1]
```diff
@@ -352,6 +352,10 @@ logo = "schreder-hyperion.png"
 url = "https://www.teamleader.eu"
 logo = "teamleader.png"
 
+[[params.users]]
+url = "https://www.theproindia.com"
+logo = "assets/images/pro-logo.svg"
```
Comment (tomkerkhove) [inline config.toml:357]: The image needs to be added to https://github.com/kedacore/keda-docs/tree/master/static/img/logos

```suggestion
logo = "pro-logo.svg"
```

**Comment by zroubalik** [comment]
@karthikeyanVK could you please update this one? thanks!

**Comment by karthikeyanVK** [comment]
struggling to check in my changes from local. Will figure out. 

image

**Comment by zroubalik** [comment]
> struggling to check in my changes from local. Will figure out.
> 
> image

You should open the PR from your fork, from the error message I can read that you are trying to push to kedacore/keda-docs repo, which won't work for you.

**Comment by karthikeyanVK** [comment]
Created a new PR, please check https://github.com/kedacore/keda-docs/pull/492

**Pull request closed without merging**